### PR TITLE
adding noUncheckedAccess to root

### PR DESCRIPTION
**Pull request by jhonny-m**

### CODE REVIEW

File: apps/crn-frontend/src/shared-research/ResearchOutput.tsx [hunk 1]
```diff
@@ -52,10 +52,12 @@ const ResearchOutput: React.FC = () => {
...
-            <TeamOutput
-              teamId={researchOutputData.teams[0]?.id}
-              researchOutputData={researchOutputData}
-            />
+            researchOutputData.teams[0]?.id && (
+              <TeamOutput
+                teamId={researchOutputData.teams[0].id}
+                researchOutputData={researchOutputData}
+              />
+            )
```
Comment (markcsims) [inline apps/crn-frontend/src/shared-research/ResearchOutput.tsx:60]: If `researchOutputData.teams[0]?.id` aren't we going to get undefined displayed in the document?

File: packages/gp2-components/src/organisms/DashboardNews.tsx [hunk 2]
```diff
@@ -81,6 +81,7 @@ const DashboardNews: React.FC<DashboardNewsProps> = ({ items }) => {
   const [expanded, setExpanded] = useState(false);
   const news = expanded ? items : items.slice(0, 1);
   const firstNews = items[0];
+  if (firstNews === undefined) return null;
```
Comment (markcsims) [inline packages/gp2-components/src/organisms/DashboardNews.tsx:84]: Doesn't `items[0]!` work here?

Comment (jhonny-m) [reply packages/gp2-components/src/organisms/DashboardNews.tsx:84]: we can't do the noNull assertion here

File: apps/crn-frontend/src/shared-research/ResearchOutput.tsx (hunk 1 above)
Comment (jhonny-m) [reply apps/crn-frontend/src/shared-research/ResearchOutput.tsx:60]: undefineds dont get rendered

Comment (jhonny-m) [reply apps/crn-frontend/src/shared-research/ResearchOutput.tsx:60]: a lot of code would be incorrect if that would be the case 😄

**Review by markcsims — APPROVED** [review]

**Pull request merged**